OtripleS.Web.Api.Tests.Unit/Services/CourseAttachments/CourseAttachmentServiceTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.Data.SqlClient;
+using Moq;
+using OtripleS.Web.Api.Brokers.DateTimes;
+using OtripleS.Web.Api.Brokers.Loggings;
+using OtripleS.Web.Api.Brokers.Storage;
+using OtripleS.Web.Api.Models.CourseAttachments;
+using OtripleS.Web.Api.Services.CourseAttachments;
+using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Runtime.Serialization;
+using Tynamix.ObjectFiller;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.CourseAttachments
+{
+    public partial class CourseAttachmentServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly Mock<IDateTimeBroker> dateTimeBrokerMock;
+        private readonly ICourseAttachmentService courseAttachmentService;
+
+        public CourseAttachmentServiceTests()
+        {
+            this.storageBrokerMock = new Mock<IStorageBroker>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+            this.dateTimeBrokerMock = new Mock<IDateTimeBroker>();
+
+            this.courseAttachmentService = new CourseAttachmentService(
+                storageBroker: this.storageBrokerMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object,
+                dateTimeBroker: this.dateTimeBrokerMock.Object);
+        }
+
+        private IQueryable<CourseAttachment> CreateRandomCourseAttachments() =>
+            CreateCourseAttachmentFiller(DateTimeOffset.UtcNow).Create(GetRandomNumber()).AsQueryable();
+
+        private static int GetRandomNumber() => new IntRange(min: 2, max: 150).GetValue();
+
```

Comment:
Don't need an empty new line between single liners as previously stated in the coding standard 

https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations